listenbrainz/db/year_in_music.py

```diff
@@ -0,0 +1,92 @@
+import psycopg2
+import sqlalchemy
+import ujson
+from flask import current_app
+from psycopg2.extras import execute_values
+
+from listenbrainz import db
+
+def get_year_in_music(user_id):
+    """ Get year in music data for requested user """
+    with db.engine.connect() as connection:
+        result = connection.execute(sqlalchemy.text("""
+            SELECT data FROM statistics.year_in_music WHERE user_id = :user_id
+        """), user_id=user_id)
+        row = result.fetchone()
+        return row["data"] if row else None
+
+
+def insert_new_releases_of_top_artists(user_id, data):
+    with db.engine.connect() as connection:
+        connection.execute(sqlalchemy.text("""
+            INSERT INTO statistics.year_in_music (user_id, data)
+                 VALUES (:user_id, jsonb_build_object('new_releases_of_top_artists', :data :: jsonb))
```

Comment:
I think it would be a good idea to document the top level keys that will be used in this field. Not sure if that should be done here or maybe in the model...

Reply:
Yes makes sense to do that. Haven't documented field yet because we have not finalized on the contents.